### PR TITLE
Tinkerforge binding and action: new devices, enhancements and fixes

### PR DESCRIPTION
**Pull request by theoweiss**
Fixes also https://github.com/openhab/openhab/issues/2968

### CODE REVIEW

File: bundles/binding/org.openhab.binding.tinkerforge/.settings/org.eclipse.jdt.core.prefs [hunk 1]
```diff
@@ -1,7 +1,11 @@
...
+org.eclipse.jdt.core.compiler.codegen.targetPlatform=1.7
+org.eclipse.jdt.core.compiler.codegen.unusedLocal=preserve
+org.eclipse.jdt.core.compiler.compliance=1.7
+org.eclipse.jdt.core.compiler.debug.lineNumber=generate
+org.eclipse.jdt.core.compiler.debug.localVariable=generate
+org.eclipse.jdt.core.compiler.debug.sourceFile=generate
 org.eclipse.jdt.core.compiler.problem.assertIdentifier=error
 org.eclipse.jdt.core.compiler.problem.enumIdentifier=error
-org.eclipse.jdt.core.compiler.source=1.6
+org.eclipse.jdt.core.compiler.source=1.7
```
Comment (teichsta) [inline bundles/binding/org.openhab.binding.tinkerforge/.settings/org.eclipse.jdt.core.prefs]: in this file you've changed to 1.7 whereas in the others you changed to 1.6

**Comment by teichsta** [comment]
this PR lgtm, except the 1.6/1.7 confusion. Could you please squash your commits into one, finally? Thanks, Thomas E.-E.

**Comment by theoweiss** [comment]
I will check the strange 1.6/1.7 confusion and hopefully master the squashing. Thanks @teichsta for the review.

**Comment by teichsta** [comment]
thanks @theoweiss!

Afaik you don't need the 1.7 requirement (and OH1.8 should generally not rely on 1.7)

**Comment by theoweiss** [comment]
I think it's better to close this PR and use https://github.com/openhab/openhab/pull/3608 as a replacement. I think squashing would raise difficulties.

**Pull request closed without merging**